cfgov/unprocessed/js/modules/util/breakpoint-state.js

```diff
@@ -4,14 +4,38 @@
 
 import varsBreakpoints from '@cfpb/cfpb-core/src/vars-breakpoints';
 
+/**
+ * @returns {number} The base font size set on the body element.
+ */
+function _getBodyBaseFontSize() {
+  return parseFloat( getComputedStyle( document.body ).fontSize );
+}
+
 /**
  * @param {Object} breakpointRange - Object containing breakpoint constants.
  * @param {integer} width - Current window width.
  * @returns {boolean} Whether the passed width is within a breakpoint range.
  */
 function _inBreakpointRange( breakpointRange, width ) {
-  const min = breakpointRange.min || 0;
-  const max = breakpointRange.max || Number.POSITIVE_INFINITY;
+  let breakpointRangeMin = breakpointRange.min;
+  let breakpointRangeMax = breakpointRange.max;
+
+  // Whether the user has set a custom size for the font in their browser.
+  const useEmsConversation = isNaN( _getBodyBaseFontSize() ) ||
+                             _getBodyBaseFontSize() === 16 ? false : true;
+  if ( useEmsConversation ) {
+    /* 16 = base font size without adjustments.
+       The CSS converts breakpoints to ems, which then change the width of the
+       pixel width of the breakpoint. In JavaScript, the breakpoints are defined
+       in pixels, so we first convert them to ems using the 16px base font size
+       and then multiply them by any adjustments set by customizations of the
+       font size in the user's browser. */
+    breakpointRangeMin = ( breakpointRangeMin / 16 ) * _getBodyBaseFontSize();
+    breakpointRangeMax = ( breakpointRangeMax / 16 ) * _getBodyBaseFontSize();
```

Comment:
L33 and L34 aren't covered by our tests (`gulp test:unit --specs=js/modules/util/breakpoint-state-spec.js`). It'd be cool if we could mock a non-standard font size in the test file by doing something like `document.body.fontSize = '22px';` but I haven't had any luck figuring it out. I would think the process would be similar to what we do with `windowResizeTo()`?

Reply:
Okay I reworked the tests in https://github.com/cfpb/cfgov-refresh/pull/5613/commits/b7765a444475da5a9a6fd7ba25a778d49ce0c29d